crates/services/txpool_v2/src/tests/pool.rs

```diff
@@ -0,0 +1,1256 @@
+#![allow(non_snake_case)]
```

Comment:
It is hard to check what tests were migrated. @AurelienFT Could you hi light that was changed mainly?

Reply:
All the pool tests were migrated, no one really had there logic changed it's just a refactoring for readibility. SOme have been added such as `insert__tx_gas_limit`, `insert__tx_bytes_limit`, `insert__tx_upgrade_with_invalid_wasm` and `insert__if_tx3_depends_and_collides_with_tx2`